string[] bin_ids
string[] asins
string[] actions
---
bool success
string message